internal/rpc/server.go

```diff
@@ -300,7 +302,18 @@ func instrumentHTTPHandler(handler http.Handler, params *ServerParams) http.Hand
 }
 
 func newGRPCServerOptions(params *ServerParams) []grpc.ServerOption {
-	opts := []grpc.ServerOption{}
+	opts := []grpc.ServerOption{
+		grpc.KeepaliveParams(keepalive.ServerParameters{
+			// MaxConnectionAge is a duration for the maximum amount of time a
```

Comment:
So will the roundrobin only happen after a minute? Will we get true roundrobin or only at chunks of this time?

Reply:
The max connection age only affects how we establish new gRPC connections, so basically instead of having a long live connection, we instruct gRPC to initialize a new connection every minute so that they can be mapped/unmapped from pods that were spawned/killed.